src/Mantasflowers.Domain/Entities/OrderAddress.cs

```diff
@@ -0,0 +1,13 @@
+namespace Mantasflowers.Domain.Entities
+{
+    public class OrderAddress : BaseEntity
```

Comment:
Common fields should be moved to an abstract class Address, whereas
```cs
public class OrderAddress : BaseEntity, Address
```
and 
```cs
public class UserAddress : BaseEntity, Address
```
could just inherit it.